docs/utilization.md

```diff
@@ -2,157 +2,201 @@
 
-    100% - a user has reached their billable goal
+Utilization | Description
+------------|-------------
+< 100% | a user has not reached their billable goal
+100% | a user has reached their billable goal
+\> 100% | a user has exceeded their billable goal
 
-    > 100% - a user has exceeded their billable goal
+### Billable hours
 
-### BILLABLE HOURS
-The sum of hours tocked to billable projects.
+The sum of hours tocked to hourly billable projects.
 
-### NON BILLABLE HOURS
-The sum of hours tocked to non-billable projects, excluding hours which were not worked like `Out of Office`.
+### Non-billable hours
 
-### TARGET BILLABLE HOURS
-A user's expected number of `BILLABLE HOURS` over a period of time. Determined by their `BILLING EXPECTATION` over that period and the total number of billable and non-billable hours tocked.
+The sum of hours tocked to hourly non-billable projects, excluding hours which were not worked like `Out of Office`.
 
-```python
-TARGET_BILLABLE_HOURS = (BILLABLE_HOURS + NON_BILLABLE_HOURS) * BILLING_EXPECTATION
-```
+### Billable allocation
+
+The sum of allocation percentages (e.g., 100%, 50%, 25%, 12.5%) tocked to weekly billable projects.
+
+### Billable allocation hours
 
-### BILLING EXPECTATION
-The percent of `TOTAL HOURS` which are expected to be billable for a given reporting period. Set per user, per timecard, and will vary as an individual's role, position, or organization change. See [the handbook](https://handbook.tts.gsa.gov/tock/#weekly-billable-hour-expectations) for additional details on billing expectations.
+The sum of hours tocked to weekly billable projects. This is a derived value based on billable allocation and target billable hours: `BILLABLE_ALLOCATION * TARGET_BILLABLE_HOURS = BILLABLE_ALLOCATION_HOURS`.
 
-### NON-BILLABLE
-An employee is considered `Non billable` if their `TARGET BILLABLE HOURS` over a time period is 0. Utilization is not calculated for these individuals because
-we're currently unable to <a href="https://en.wikipedia.org/wiki/Division_by_zero">divide by zero</a>.
+**Note:** Allocation percentages technically refer to an employee's availability on a project rather than a specific number of hours worked. But we convert this percentage to an _expected_ hourly value to enable utilization rate calculations across durations (e.g., the past month) and groups (e.g., all of `Project ABC`) that may include varying combinations of hourly, weekly, and excluded hours projects.
 
-All `BILLABLE HOURS` tocked by `Non billable` employees are included in aggregations for their respective business unit and organization.
+### Target billable hours
+A user's expected number of hours tocked to billable projects over a period of time. Determined by their [billing expectation](#billing-expectation) over that period and the total number of hours in a week less any hours which were not worked like `Out of office`.
```

Comment:
[question] Are you using `TARGET_BILLABLE_HOURS` and `BILLING_EXPECTATION` instead of the timecard field names (`target_hours` and `billable_expectation`) to match what was already in the documentation?

Reply:
That makes sense to me!